durable-task-step/src/main/resources/org/jenkinsci/plugins/workflow/steps/durable_task/ShellStep/help.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    Runs a Bourne shell script on a Unix node using <code>nohup</code>. Multiple lines are accepted.
```

Comment:
The `nohup` aspect is probably confusing and irrelevant for most users, and anyway might become inaccurate if I ever get to [JENKINS-25503](https://issues.jenkins-ci.org/browse/JENKINS-25503).